tests/test_densepose.py

```diff
@@ -60,7 +60,7 @@ def test_image(model, chimp_image_path, tmp_path):
         )
 
         # output to disk
-        assert anatomy_info.shape == (2, 44)
+        assert anatomy_info.shape == (1, 44)
```

Comment:
unrelated to caching, this is due to new densepose model release: https://github.com/facebookresearch/detectron2/tree/main/projects/DensePose#whats-new